lib/graphql/metrics/trace.rb

```diff
@@ -133,7 +133,7 @@ def capture_parsing_time
       def capture_validation_time(context)
         # Queries may already be lexed and parsed before execution (whether a single query or multiplex).
         # If we don't have those values, use some sane defaults.
-        if pre_context.lexing_duration.nil?
+        if [nil, 0].include?(pre_context.lexing_duration)
           pre_context.lexing_start_time_offset = pre_context.multiplex_start_time
           pre_context.lexing_duration = 0.0
         end
```

Comment:
Wondering if we should just skip all this conditional for lexing/parsing and default the durations to `0.0` to start with and fallback to `multiplex_start_time` if neither of the start times exist.

Then the test helper wouldn't be needed either?